src/platforms/node/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -28,7 +30,11 @@ Sentry.init({
     // enable HTTP calls tracing
     new Sentry.Integrations.Http({ tracing: true }),
     // Automatically instrument Node.js libraries and frameworks
+    // (This might degrade startup performance)
```

Comment:
Idk about this line. It just says it may (100% will) degrade performance and there's no clear indicator of what the alternative is.

Maybe a slight reword?:

```suggestion
    // (This function is slow because of file I/O, consider manually adding additional integrations instead)
```